qwerty_pyrt/rust/src/wrap_repl.rs

```diff
@@ -27,4 +27,6 @@ impl ReplState {
     fn run(&self, expr: Expr) -> Expr {
         Expr::new(self.state.lock().unwrap().run(&expr.expr))
     }
+
 }
+
```

Comment:
I would be grateful if we could remove these blank lines:
```suggestion
}
```